books/RayTracingInOneWeekend.html

```diff
@@ -1265,20 +1271,91 @@
         return (1.0-t)*vec3(1.0, 1.0, 1.0) + t*vec3(0.5, 0.7, 1.0);
     }
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [ray-color-depth]: <kbd>[main.cc]</kbd> ray_color() with depth limiting]
+    [Listing [ray-color-unit-sphere]: <kbd>[main.cc]</kbd> ray_color() with replacement diffuse]
 </div>
 
 <div class='together'>
 After rendering we get a similar image:
 
-  ![Figure [correct-lambertian-sphere]: Correct rendering of a diffuse sphere](../images/img.correct-lambertian.jpg)
+  ![Correct rendering of Lambertian spheres]: ](../images/img.correct-lambertian.png)
 
 It's hard to tell what the difference between these two diffuse methods are, given that our scene of
```

Comment:
It's hard to tell ~what~ the difference between these two diffuse methods ~are~, given ...